### PR TITLE
Update resource page and contact page

### PR DESCRIPTION
**Pull request by sijia-w**
Motivation:
The resource page is out of date.

Modification:
1. The tech blog of the same series are put next to each other.
2. The shared slides are updated.
3. The Bot WeChat ID is added on contact page.

### CODE REVIEW

**Comment by sijia-w** [comment]
@Jennifer88huang Please help review the changes. But I am not sure about what to change on the Event Page. Please leave some hints here. Thank you.

**Comment by sijia-w** [comment]
@Jennifer88huang  Oops, everything was up to date. I have no idea why the unit test fails again. And I tried the commands you showed me last time, still everything up to date. Quite confused.

File: site2/website/pages/en/contact.js [hunk 1]
```diff
@@ -88,6 +88,11 @@ class Contact extends React.Component {
 
             <p><translate>You can self-register at </translate> <a href="https://apache-pulsar.herokuapp.com/" target="_blank">
                     https://apache-pulsar.herokuapp.com/</a></p>
+
+            <h2><translate>WeChat</translate></h2>
+            <p><translate>There are several WeChat groups that are used for informal discussions for devs and users.</translate></p>
+
+            <p><translate>You can add Bot by the WeChat ID: StreamNative_BJ</translate></p>
```
Comment (Jennifer88huang) [inline site2/website/pages/en/contact.js:95]:
```suggestion
            <p><translate>To join those WeChat tech groups, you can add Bot with the WeChat ID: StreamNative_BJ</translate></p>
```

**Comment by Jennifer88huang** [comment]
@sijia-w I notice that you've refined the `SinksBase.java` file, do we need to update this file?

**Comment by sijia-w** [comment]
@Jennifer88huang  Thanks for pointing this out. I did not even git add that file. So strange.

**Pull request closed without merging**